Codeinwp/otter-blocks
# Add Inspector notice when reCaptcha is active.

Closes #1574.


### Summary


Added a little notice to inform the user about the location of the API Keys.

### Screenshots 

![image](https://user-images.githubusercontent.com/17597852/230379861-a4fa752e-f5c4-4a37-b3cd-fb8e7740b1b6.png)


----

### Test instructions


ℹ️ API Keys are optional for testing this. You can use any value when it asks to introduce them. In Editor, we are just checking if the values are set.

1. Create a form block.
2. Activate the reCaptcha in Inspector.
3. Check the Inspector for the notice.


https://user-images.githubusercontent.com/17597852/230382084-75a68ef9-6eee-4739-9b30-bebd9711e5d8.mp4





---- 

### Checklist before the final review
